base_view_inheritance_extension/models/ir_ui_view.py

```diff
@@ -55,6 +55,9 @@ def _iter_inheritance_specs(self, spec):
                     yield node, handler
             return
         if spec.get('position') == 'attributes':
+            if all(not c.get('operation') for c in spec):
```

Comment:
What if one has operation and another doesn't?

Reply:
then it's a view which uses this module and then we can expect devs to order the child nodes in a way it doesn't break. Might be a good idea to talk about that in the readme.
The important issue this PR is meant to fix is that just installing this module breaks completely fine code like https://github.com/OCA/OCB/blob/10.0/addons/product/views/product_views.xml#L285